homeassistant/helpers/entity_platform.py

```diff
@@ -136,6 +136,7 @@ def __init__(
         self._process_updates: asyncio.Lock | None = None
 
         self.parallel_updates: asyncio.Semaphore | None = None
+        self._update_in_parallel: bool = True
```

Comment:
This will mean we always update in sequence even when there's no semaphore.

Reply:
Thanks. It looks like I messed something up when I separated this out of my integration branch.  Will clean it up and retest shortly

Reply:
https://github.com/home-assistant/core/pull/93693